airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -80,30 +84,26 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
     final int version = Workflow.getVersion(VERSION_LABEL, Workflow.DEFAULT_VERSION, CURRENT_VERSION);
     final String taskQueue = Workflow.getInfo().getTaskQueue();
 
-    // Temporarily suppressed to address OC issue #1210
-    @SuppressWarnings("PMD.UnusedLocalVariable")
     final int autoDetectSchemaVersion =
         Workflow.getVersion(AUTO_DETECT_SCHEMA_TAG, Workflow.DEFAULT_VERSION, AUTO_DETECT_SCHEMA_VERSION);
 
-    // Temporarily disabled to address OC issue #1210
-    // if (autoDetectSchemaVersion >= AUTO_DETECT_SCHEMA_VERSION) {
-    // final Optional<UUID> sourceId = configFetchActivity.getSourceId(connectionId);
-    //
-    // if (!sourceId.isEmpty() && refreshSchemaActivity.shouldRefreshSchema(sourceId.get())) {
-    // LOGGER.info("Refreshing source schema...");
-    // refreshSchemaActivity.refreshSchema(sourceId.get(), connectionId);
-    // }
-    //
-    // final Optional<Status> status = configFetchActivity.getStatus(connectionId);
-    // if (!status.isEmpty() && Status.INACTIVE == status.get()) {
-    // LOGGER.info("Connection is disabled. Cancelling run.");
-    // final StandardSyncOutput output =
-    // new StandardSyncOutput()
-    // .withStandardSyncSummary(new
-    // StandardSyncSummary().withStatus(ReplicationStatus.CANCELLED).withTotalStats(new SyncStats()));
-    // return output;
-    // }
-    // }
+    if (autoDetectSchemaVersion >= AUTO_DETECT_SCHEMA_VERSION) {
```

Comment:
I think that we will need a new version here. Since it wasn't commented out, the version will be in the history and even old workflow will run into undeterministic exception.

Reply:
I updated AUTO_DETECT_SCHEMA_VERSION to 2. is that what you mean?

Reply:
Yes that will work